graylog2-server/src/main/java/org/graylog2/bootstrap/ServerBootstrap.java

```diff
@@ -304,7 +304,15 @@ public void runMigrations() {
 
         ImmutableSortedSet.copyOf(migrations).forEach(m -> {
             LOG.debug("Running migration <{}>", m.getClass().getCanonicalName());
-            m.upgrade();
+            try {
+                m.upgrade();
+            } catch (Exception e) {
+                if (configuration.ignoreMigrationFailures()) {
+                    LOG.warn("  Ignoring migration failure: {}", e.getMessage());
```

Comment:
```suggestion
                    LOG.warn("Ignoring migration failure: {}", e.getMessage());
```
Remove leading whitespace

Reply:
only when debugging is turned on, though

Reply:
Oops - missed that. Then it would also be good to include the migration name in the message.